docs/deploy/index.md

```diff
@@ -121,7 +122,16 @@ Now, forward a local port to the ingress controller:
 kubectl port-forward --namespace=ingress-nginx service/ingress-nginx-controller 8080:80
 ```
 
-At this point, if you access http://demo.localdev.me:8080/, you should see an HTML page telling you "It works!".
+!!! A note on DNS & network-connection
+
+  This documentation assumes that a user has awareness of the DNS and the network routing aspects involved in using ingress.
+  The port-forwarding mentioned above, is the easiest way to demo the working of ingress.
+  Port-forwarding is not recommended way for a production environment use-case.
+  [This issue](https://github.com/kubernetes/ingress-nginx/issues/10014#issuecomment-1567791549described) shows a typical DNS problem and its solution 
+
+At this point, you can access your deployment in 2 ways ;
+- Use curl like `curl http://demo.localdev.me:8080`
```

Comment:
```suggestion
- Use curl like `curl --resolve demo.localdev.me:8080:127.0.0.1 http://demo.localdev.me:8080`
```

I think we can directly use this command so that users don't need to know how to set up the hosts record and other related steps.

Reply:
@tao12345666333 I changed as per suggestion. Please review again